pkg/storage/stores/shipper/bloomshipper/client.go

```diff
@@ -68,7 +68,7 @@ type MetaClient interface {
 	// Returns all metas that are within MinFingerprint-MaxFingerprint fingerprint range
 	// and intersect time period from StartTimestamp to EndTimestamp.
 	GetMetas(ctx context.Context, params MetaSearchParams) ([]Meta, error)
-	PutMeta(ctx context.Context, meta Meta) (Meta, error)
+	PutMeta(ctx context.Context, meta Meta) error
```

Comment:
@vlad-diachenko the `PutObject` method that is called in the implementation of `PutMeta` does not return the object back. Are you planning to fetch the object after putting? 
I removed the Meta from return value to make life easier for this PR. Hope that's ok.